components/date-time/date.js

```diff
@@ -13,12 +13,16 @@ function DatePicker( { currentDate, onChange, ...args } ) {
 	const momentDate = currentDate ? moment( currentDate ) : moment();
 	const onChangeMoment = ( newDate ) => onChange( newDate.format( TIMEZONELESS_FORMAT ) );
 
-	return <ReactDatePicker
-		inline
-		selected={ momentDate }
-		onChange={ onChangeMoment }
-		{ ...args }
-	/>;
+	return (
+		<div className="datepicker-container" tabIndex="-1" aria-hidden="true">
```

Comment:
Not sure `tabIndex="-1"` will help so much, as it works just on the element it's applied to. This div is already non-focusable. Instead, I'm still able to tab to the Prev and Next buttons in the date picker.

Reply:
I'm looking into this right now. But alas, this also depends on the date picker component itself. I just sent in a PR for React-Datepicker that would introduce text value to the buttons as that seems to be the only way to get this to work without forking the date picker (not a good idea) or using the onFocus hook to disable those buttons.